### PR TITLE
1.0 RC

### PR DESCRIPTION
**Pull request by mafintosh**
Mainly includes

``` js
db.get(key, cb)
db.put(key, value, cb)
```

Seems to perform *much* faster than the current master.

Goals for 1.1 are `db.del`, `db.watch`, `db.iterator` and `db.createReadStream/createWriteStream`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,35 +1,27 @@
...
+    "tape": "^4.8.0"
   },
   "scripts": {
-    "test": "standard && tape test.js"
+    "test": "tape test/*.js"
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/mafintosh/hyperdb.git"
+    "url": "git+https://github.com/mafintosh/hyperdb-dev.git"
```
Comment (ralphtheninja) [inline package.json]: Private repository?

File: package.json [hunk 2]
```diff
@@ -1,35 +1,27 @@
...
   "devDependencies": {
-    "hypercore": "^6.3.6",
-    "random-access-memory": "^2.4.0",
-    "run-series": "^1.1.4",
-    "standard": "^10.0.2",
-    "tape": "^4.6.3"
+    "tape": "^4.8.0"
   },
   "scripts": {
-    "test": "standard && tape test.js"
```
Comment (ralphtheninja) [inline package.json]: Y u no standard? :)

File: package.json (hunk 1 above)
Comment (mafintosh) [inline package.json]: nice catch

File: package.json (hunk 2 above)
Comment (mafintosh) [inline package.json]: fixed

File: index.js [hunk 3]
```diff
@@ -1,604 +1,376 @@
...
+    if (!h.length) {
+      self.writers[log].append({
+        seq: self.writers[log].feed.length,
+        log: log,
+        key: key,
+        path: path,
+        value: value,
+        heads: newHeads,
+        trie: []
+      }, cb)
```
Comment (emilbayes) [inline index.js]: Would be nice with:

```js
var node = {
  seq: self.writers[log].feed.length,
  log: log,
  key: key,
  path: path,
  value: value,
  heads: newHeads,
  trie: []
}
self.writers[log].feed.append(function (err) {
  if (err) return cb(err)
  return cb(node)
})
```

Likewise in `onput`

Comment (emilbayes) [inline index.js]: https://github.com/mafintosh/hyperdb/pull/5

**Pull request merged**